app/controllers/api/v1/articles_controller.rb

```diff
@@ -0,0 +1,65 @@
+class Api::V1::ArticlesController < ApplicationController
+  def index
+    articles = if params.key?('page')
+                 Article.all.limit(3).offset(3 * (params[:page].to_i - 1))
+               else
+                 Article.all
+               end
+    paginate articles, per_page: 3
+    render json: articles, status: 200
+  end
+
+  def show
+    article = Article.find(params[:id])
```

Comment:
use find_by method to handle cases where the article is not found without raising an exception